docs/preview/features/powershell/azure-api-management.md

```diff
@@ -24,17 +24,17 @@ PS> Import-Module -Name Arcus.Scripting.ApiManagement
 
 Create an operation on an existing API in Azure API Management.
 
-| Parameter           | Mandatory | Description                                                                                              |
-| ------------------- | --------- | -------------------------------------------------------------------------------------------------------- |
-| `ResourceGroupName` | yes       | The resource group containing the Azure API Management instance                                          |
-| `ServiceName`       | yes       | The name of the Azure API Management instance located in Azure                                           |
-| `ApiId`             | yes       | The ID to identify the API running in Azure API Management                                               |
-| `OperationId`       | yes       | The ID to identify the to-be-created operation on the API                                                |
-| `Method`            | yes       | The method of the to-be-created operation on the API                                                     |
-| `UrlTemplate`       | yes       | The URL-template, or endpoint-URL, of the to-be-created operation on the API                             |
-| `OperationName`     | no        | The optional descriptive name to give to the to-be-created operation on the API (default: `OperationId`) |
-| `Description`       | no        | The optional explanation to describe the to-be-created operation on the API                              |
-| `PolicyFilePath`    | no        | The path to the file containing the optional policy of the to-be-created operation on the API            |
+| Parameter        | Mandatory | Description                                                                                              |
+| ---------------- | --------- | -------------------------------------------------------------------------------------------------------- |
+| `ResourceGroup`  | yes       | The resource group containing the Azure API Management instance                                          |
```

Comment:
Is there a reason why we are renaming these? Because our other commands are actually using `ResourceGroupName`, not `ResourceGroup`

Reply:
I get why Stijn is changing this, apparently, for the other APIM-scripts we used "ResourceGroup" instead of "ResourceGroupName" as well. 
Only Storage.Table also uses "ResourceGroup", all others (where applicable) use "ResourceGroupName". 
We should be consistent, so the earlier we align everything to "ResourceGroupName", the better. (#95)

Reply:
What he said 😃 .